src/deroff.rs

```diff
@@ -840,6 +840,22 @@ impl Deroffer {
             None => panic!("do_line` called with empty string as argument"),
         }
     }
+
+    fn str_eq(&self, offset: usize, other: &str, len: usize) -> bool {
```

Comment:
Can remove this guy now. :disappointed: womp womp

Reply:
Removed.

Reply:
This guy is not used in other functions.